### PR TITLE
[Admin] Save data before update, create

### PR DESCRIPTION
**Pull request by at-khadinh**
## Description
Save data before create, update product.
## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: resources/views/admin/pages/products/edit.blade.php [hunk 1]
```diff
@@ -86,7 +86,7 @@
                 @foreach($product->images as $image)
```
Comment (at-mailuong) [inline resources/views/admin/pages/products/edit.blade.php]: space nha

File: resources/views/admin/pages/products/index.blade.php [hunk 2]
```diff
@@ -140,6 +140,10 @@
     </div>
   </div>
 </div>
+<script src="/js/messages.js"></script>
 <script src="/js/product.js"></script>
+@endsection
+@section('css')
```
Comment (at-mailuong) [inline resources/views/admin/pages/products/index.blade.php]: js nha

**Comment by at-mailuong** [comment]
LGTM

**Comment by at-mailuong** [comment]
LGTM

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**